steward/src/commands.rs

```diff
@@ -85,13 +85,22 @@ impl Configurable<StewardConfig> for EntryPoint {
             .map(PathBuf::from)
             .unwrap_or_else(|| CONFIG_FILE.into());
 
-        if filename.exists() {
+        let config_env_variable = option_env!("STW_ENV_VAR");
```

Comment:
We don't need to say "ENV_VAR", it's implicit. `STEWARD_CONFIG` would be preferable.

Reply:
Additionally, `option_env!` is a compile-time macro. We can check for the presence of runtime env vars using `std::env::var("VARIABLE").is_ok()`.

Reply:
done